scaleway/resource_lb_backend_beta.go

```diff
@@ -105,6 +107,98 @@ func resourceScalewayLbBackendBeta() *schema.Resource {
 				ValidateFunc:     validateDuration(),
 				Description:      "Maximum tunnel inactivity time",
 			},
+
+			// Health Check
+			"health_check_timeout": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateDuration(),
+				Default:      "30s",
+				Description:  "Timeout before we consider a HC request failed",
+			},
+			"health_check_delay": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateDuration(),
+				Default:      "60s",
+				Description:  "Interval between two HC requests",
+			},
+			"health_check_port": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Computed:    true,
+				Description: "Port the HC requests will be send to. Default to `forward_port`",
+			},
+			"health_check_max_retries": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Default:     2,
+				Description: "Number of allowed failed HC requests before the backend server is marked down",
+			},
+			"health_check_tcp": {
+				Type:          schema.TypeList,
+				MaxItems:      1,
+				ConflictsWith: []string{"health_check_http", "health_check_https"},
+				Optional:      true,
+				Computed:      true,
```

Comment:
why is this one computed and not `health_check_http` & `health_check_https`?

Reply:
Because if none are specified Api default to TCP. So in some case it can be computed